tests/unit_tests/test_game_rating.py

```diff
@@ -250,12 +282,17 @@ async def test_on_game_end_ladder_ratings_persisted(ladder_game, players):
     await ladder_game.on_game_end()
     await rating_service._join_rating_queue()
 
-    results = get_persisted_results(rating_service)
-    assert results.rating_type == RatingType.LADDER_1V1
-    assert players.hosting.id in results.ratings
-    assert players.joining.id in results.ratings
-    assert results.outcomes[players.hosting.id] is GameOutcome.VICTORY
-    assert results.outcomes[players.joining.id] is GameOutcome.DEFEAT
+    ladder_results, global_results = get_persisted_results(rating_service)
+    assert ladder_results.rating_type == RatingType.LADDER_1V1
+    assert players.hosting.id in ladder_results.ratings
+    assert players.joining.id in ladder_results.ratings
+    assert ladder_results.outcomes[players.hosting.id] is GameOutcome.VICTORY
+    assert ladder_results.outcomes[players.joining.id] is GameOutcome.DEFEAT
+
+    assert global_results.rating_type == RatingType.GLOBAL
+    assert players.hosting.id in global_results.ratings
+    assert players.joining.id not in global_results.ratings
+    assert global_results.outcomes[players.hosting.id] is GameOutcome.VICTORY
```

Comment:
What about the defeat for the second player?

Reply:
The defeated player's rating adjustment would have caused the displayed rating to decrease therefore it should not be applied.

Reply:
Can we get a comment there explaining this?

Reply:
.

Reply:
Oh yea, missed it since it was from previous review. Do we really need a comment though because it already has the line  
```python
 assert players.joining.id not in global_results.ratings
```
asserting that the rating of the second player should not change. I think there are a number of tests that all have this same behavior

Reply:
Oh yeah, when I first looked at it I didn't notice the `not`. I guess it is ok